src/SFA.DAS.EmployerAccounts.Web/Mappings/AgreementMappings.cs

```diff
@@ -35,6 +35,14 @@ public AgreementMappings()
 
             CreateMap<AccountDetailViewModel, AgreementInfoViewModel>()
                 .ConvertUsing(new AgreementInfoConverter());
+
+            CreateMap<EmployerAgreementDto, OrganisationAgreementViewModel>()
+                .ForMember(dest => dest.SignedDateText, opt => opt.MapFrom(src => src.SignedDate.HasValue ? src.SignedDate.Value.ToString("dd MMMM yyyy") : ""))
```

Comment:
This logic could just be a get property rather than hidden in a mapper

Reply:
done